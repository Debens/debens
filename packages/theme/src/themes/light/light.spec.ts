import { light } from './light';

describe('light theme', () => {
    it('then should not have any unexpected changes', () => {
        expect(light).toMatchSnapshot();
    });
});
