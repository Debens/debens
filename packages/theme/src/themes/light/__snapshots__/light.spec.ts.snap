// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`light theme then should not have any unexpected changes 1`] = `
Object {
  "color": Object {
    "$background-active": "",
    "$background-brand": "",
    "$background-primary": "",
    "$button-danger": "",
    "$button-danger-active": "",
    "$button-danger-hover": "",
    "$button-disabled": "",
    "$button-primary": "",
    "$button-primary-active": "",
    "$button-primary-hover": "",
    "$button-secondary": "",
    "$button-secondary-active": "",
    "$button-secondary-hover": "",
    "$ghost": "",
    "$layer-01": "",
    "$layer-02": "",
    "$layer-03": "",
    "$layer-active-01": "",
    "$layer-active-02": "",
    "$layer-active-03": "",
    "$layer-disabled-01": "",
    "$layer-disabled-02": "",
    "$layer-disabled-03": "",
    "$layer-hover-01": "",
    "$layer-hover-02": "",
    "$layer-hover-03": "",
    "$layer-selected-01": "",
    "$layer-selected-02": "",
    "$layer-selected-03": "",
    "$text-disabled": "",
    "$text-error": "",
    "$text-on-color": "",
    "$text-placeholder": "",
    "$text-primary": "",
    "$text-secondary": "",
  },
  "radii": Object {
    "large": 16,
    "medium": 8,
    "none": 0,
    "small": 4,
    "xlarge": 24,
    "xsmall": 2,
  },
  "spacing": Object {
    "large": 16,
    "medium": 8,
    "none": 0,
    "small": 4,
    "xlarge": 24,
    "xsmall": 2,
  },
  "typography": Object {
    "$body": Object {
      "family": "// TODO: unknown",
      "height": 12,
      "size": 12,
      "weight": 600,
    },
    "$heading": Object {
      "family": "// TODO: unknown",
      "height": 12,
      "size": 12,
      "weight": 600,
    },
  },
}
`;
